onnx/backend/test/case/node/split.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Split(Base):
+
+    @staticmethod
+    def export_1d():
+        input = np.array([1., 2., 3., 4., 5., 6.])
```

Comment:
Please explicitly specify the dtype=float here.

Reply:
fixed